app/content/models/strike.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+
+from django.db import models
+
+from app.content.models import Event, User
+from app.util.models import BaseModel
+from app.util.utils import today
+
+
+class Strike(BaseModel):
+    id = models.UUIDField(
+        auto_created=True, primary_key=True, default=uuid.uuid4, serialize=False,
+    )
+    description = models.CharField(max_length=200)
+    expires_at = models.DateTimeField()
```

Comment:
Er det ikke bedre å bruke created_at også heller regne ut utløpsdato med en konstant? Da blir det mye enklere å evt endre varighet på prikker i fremtiden.

Reply:
Mener det er bedre å bruke expires_at. To hovedgrunnen er at da slipper man dato manipulasjon etter at en prikk er laget. Er bare å teste om den har gått ut på dato, som er en fordel siden dato er pain å drive med. I tillegg vil dette åpne muligheten til ulike "utløpskonstanter" der vi bruker disse konstantene når vi lager en prikk. Dette fører til mer samkjøring frontend og backend, siden konstantene vil kun ligge i backend da.

Reply:
Om du mener at expires_at burde være en @property kan jeg gå med på den tanken. Vil være litt hip som happ, men vi vil miste mulighet til å "endre" på denne datoen i admin panelet dersom det noen gang trengs

Reply:
Litt det samme ja, føler vel egt bare at det er bedre å ha en "dynamisk" expries_at ift en "hardkodet" en. Synes ikke heller at det bør være mulig å endre sluttdatoen på en opprettet prikk